package.json

```diff
@@ -87,7 +87,7 @@
   },
   "dependencies": {
     "@coder/logger": "^3.0.0",
-    "argon2": "0.29.0",
+    "argon2": "0.30.2",
```

Comment:
This should be okay. They fixed the 0.30.1 issue I raised here: https://github.com/ranisalt/node-argon2/issues/362